packages/calcite-components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -126,6 +143,20 @@ export class RadioButtonGroup {
     }
   };
 
+  private getFocusableRadioButton(): HTMLCalciteRadioButtonElement | null {
+    let index = 0;
+    let focusableEle = this.radioButtons[index];
```

Comment:
For the variable name, can it be `focusableEl` instead of `focusableEle` to be consistent with other variable names?

Reply:
yup it should be `focusableEl`.